packages/medusa/src/api/routes/store/order-edits/complete-order-edit.ts

```diff
@@ -55,12 +59,22 @@ export default async (req: Request, res: Response) => {
   const orderEditService: OrderEditService =
     req.scope.resolve("orderEditService")
 
+  const orderService: OrderService = req.scope.resolve("orderService")
+
+  const paymentProviderService: PaymentProviderService = req.scope.resolve(
+    "paymentProviderService"
+  )
+
   const manager: EntityManager = req.scope.resolve("manager")
 
   const userId = req.user?.customer_id ?? req.user?.id ?? req.user?.userId
 
   await manager.transaction(async (manager) => {
     const orderEditServiceTx = orderEditService.withTransaction(manager)
+    const orderServiceTx = orderService.withTransaction(manager)
+    const paymentProviderServiceTx =
+      paymentProviderService.withTransaction(manager)
+
     const orderEdit = await orderEditServiceTx.retrieve(id)
```

Comment:
todo: join payment collection
```suggestion
    const orderEdit = await orderEditServiceTx.retrieve(id, { relations: ["payment_collection"] })
```